softwarereview_editor.Rmd

```diff
@@ -8,74 +8,72 @@ aliases:
+- Regularly (for instance weekly) monitor the pace of all open reviews by keeping an eye on the [*Dashboard* page](https://dashboard.ropensci.org/reviews.html), and reminding other editors to move packages along as needed.
 
-- Making decisions on scope and overlap for pre-submission inquiries, referrals from JOSS or other publication partners, and submissions. Discussions should be initiated in the rOpenSci Slack editors-only channel through summarising the (pre-)submitted/referred software, along with any concerns the EiC might have. If after the EiC feels they haven't received enough answers after a day or two, they can ping all editors.
+- Respond where appropriate to issues posted to the [software-review-meta repo](https://github.com/ropensci/software-review-meta/issues?q=sort%3Aupdated-desc%20is%3Aissue%20is%3Aopen).
 
-  - Any editor should feel free to step in on these. See [this section](#outofscoperesponse) about how to respond to out-of-scope (pre-) submissions.
-  - After explaining an out-of-scope decision, write an issue comment `@ropensci-review-bot out-of-scope`.
+### EiC duties for each initial submission {#eic-submission}
 
-### The rOpenSci Editorial Dashboard {#eic-dashboard}
+The EiC is responsible for initial processing of all new submissions.
+The primary duties are:
 
-The [*rOpenSci Editorial Dashboard*](https://dashboard.ropensci.org) is updated daily, primarily by extracting information on all software review issues on GitHub, along with additional information from Slack and our Airtable database.
-The dashboard provides an up-to-date overview of our editorial team, their recent responsibilities, and the current state of all software review issues.
-The EiC (or any editors who are interested) can gain an overview of the editorial team status, availability, and recent workloads on [the *editors* page](https://dashboard.ropensci.org/editors.html).
-This should be used to find and assign editors for new software review issues.
-An overview of all current software reviews is on [the **Software Review* page](https://dashboard.ropensci.org/reviews.html).
-Entries on this page are colored by a measure of "urgency", summarised in the [table at the bottom of that page](https://dashboard.ropensci.org/reviews.html#urgency-of-reviews).
+1. To decide whether or not a submission should be considered in scope and proceed to review, and if so,
+2. To proceed to a full submission and assign a handling editor.
 
-Specific tasks for reviews in the specific review stages include:
+#### Deciding on scope and overlap
 
-- Looking over submissions in "0\/presubmission" and "1\/editorial-team-prep", to check whether any action needs to be taken (such as polling editors, making decisions, putting issues on hold, pinging for updates, or finding and assigning editors).
+The EiC is entitled to take scope and overlap decisions as independently as possible, but is recommended to request opinions on the `#editors-only` channel on Slack.
```

Comment:
Add that one should request opinions for submissions outside of one's areas of expertise.